src/Silverback.Integration/Messaging/Configuration/ProducerEndpointBuilderEncryptUsingExtensions.cs

```diff
@@ -28,13 +28,17 @@ public static class ProducerEndpointBuilderEncryptUsingExtensions
         ///     The optional initialization vector (IV) for the symmetric algorithm. If <c>null</c> a different IV
         ///     will be generated for each message and prepended to the actual message payload.
         /// </param>
+        /// <param name="keyIdentifier">
+        ///     The key identifier.
+        /// </param>
         /// <returns>
         ///     The endpoint builder so that additional calls can be chained.
         /// </returns>
         public static TBuilder EncryptUsingAes<TBuilder>(
```

Comment:
I think that we should add another overload to allow a clean call like `EncryptUsingAes(myKey, myIdentifier)`, without having to pass an extra `null`.
Plus, modifying the signature of an existing method can be a breaking change.

Reply:
The overload have been added.